examples/BasicOperations/GetCampaigns.php

```diff
@@ -98,7 +98,7 @@ public static function runExample(GoogleAdsClient $googleAdsClient, $customerId)
         // Issues a search request by specifying page size.
         $response =
             $googleAdsServiceClient->search($customerId, $query, ['pageSize' => self::PAGE_SIZE]);
-
+        
```

Comment:
Nit: extra spaces

Reply:
Fixed, thanks.

Reply:
It's not fixed yet. There should be no spaces in this line at all. Should be just a blank line, like it's never modified.